client/coniksclient/internal/cmd/run.go

```diff
@@ -17,6 +17,10 @@ const help = "- register [name] [key]:\r\n" +
 	"	Register a new name-to-key binding on the CONIKS-server.\r\n" +
 	"- lookup [name]:\r\n" +
 	"	Lookup the key of some known contact or your own bindings.\r\n" +
+	"- enable timestamp:\r\n" +
+	"	Print timestamp of format <15:04:05.999999999> along with the result.\r\n" +
+	"- disable timestamp:\r\n" +
+	"	Disable timestamp printing.\r\n" +
```

Comment:
I still think a debug flag would be more elegant, but at the same time, I also kinda like the option to toggle on/off the timestamp at runtime.